app/jobs/tracer_job.rb

```diff
@@ -1,8 +1,11 @@
-class TracerJob < ApplicationJob
-  queue_as :default
-  self.queue_adapter = :sidekiq
+class TracerJob
+  include Sidekiq::Worker
+  sidekiq_options retry: false # job will be discarded if it fails
```

Comment:
...because we don't want failed jobs here to retry, because these are expected errors.